### ISSUE TITLE
Add 318MHz with 9bit

### ISSUE THREAD

**Issue by hotdjdave** [tobiabocchi/flipperzero-bruteforce#27]
In the USA, there are several garage/gate opener remotes that use 318MHz with 9bit/pin.  Is there a way to add this protocol to the bruteforce apps to be in the next update (Unleashed and others)?

I tried to make this a feature request, but that option does not seem to be available.

Thanks.

**Comment by Trip-Van-Winkle** [comment]
Also onboard with this query...?

**Comment by tobiabocchi** [comment]
Yes this should not take long, could you please provide a .sub of the raw recording of the signal?
How does flipper recognize the signal (if it does) when using "read"?
